### ISSUE TITLE
[Docs] Trying to find the definition of mmdet.CrossEntropyLoss

### ISSUE THREAD

**Issue by JBKnights** [open-mmlab/mmdetection3d#2742]
### Branch

main branch  https://mmdetection3d.readthedocs.io/en/latest/

### 📚 The doc issue

I'm trying to adjust the Cross Entropy Loss as implemented in mmdet3d; however, when I look in mmdet.losses.__init__.py it does not seem to be defined or imported from here, and I can't track down where it is defined in the code base.  Could I get some help finding the source for the mmdet3d of CE?
